cpp/src/scalar/scalar.cpp

```diff
@@ -56,4 +57,9 @@ std::string string_scalar::to_string(rmm::cuda_stream_view stream) const
   return result;
 }
 
+list_scalar::value_type list_scalar::value(rmm::cuda_stream_view stream) const
+{
+  CUDF_FAIL("cudf::list_view not yet implemented.");
```

Comment:
Is this a WIP PR? If so, please convert it to a draft PR.

Reply:
`list_view` currently has no implementation: https://github.com/rapidsai/cudf/blob/branch-0.19/cpp/include/cudf/lists/list_view.cuh

Should we return `list_view{}` here?